rotkehlchen/data_import/importers/bitcoin_tax.py

```diff
@@ -0,0 +1,303 @@
+import csv
+import hashlib
+import logging
+from pathlib import Path
+from typing import Any, Literal, Optional
+
+from rotkehlchen.accounting.structures.balance import AssetBalance, Balance
+from rotkehlchen.accounting.structures.base import HistoryEvent
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.assets.converters import LOCATION_TO_ASSET_MAPPING, asset_from_common_identifier
+from rotkehlchen.constants import ZERO
+from rotkehlchen.data_import.utils import BaseExchangeImporter, UnsupportedCSVEntry
+from rotkehlchen.db.drivers.gevent import DBCursor
+from rotkehlchen.errors.asset import UnknownAsset
+from rotkehlchen.errors.misc import InputError
+from rotkehlchen.errors.serialization import DeserializationError
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.serialization.deserialize import (
+    deserialize_asset_amount,
+    deserialize_timestamp_from_date,
+)
+from rotkehlchen.types import Fee, Location, TimestampMS
+from rotkehlchen.utils.misc import ts_sec_to_ms
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+
+class BitcoinTaxImporter(BaseExchangeImporter):
+    CSVType = Literal['trades', 'other_events']
+
+    ACTION_TO_HISTORY_EVENT_TYPE = {
```

Comment:
Sorry, just noticed, but we tend to defined constants at the module level and not in classes. Same with static methods